src/RulesTest/Library/ControlShouldSupportTablePattern.cs

```diff
@@ -52,5 +44,31 @@ public void CheckPropertyIdIsSet()
         {
             Assert.AreEqual(PropertyType.UIA_IsTablePatternAvailablePropertyId, this.Rule.Info.PropertyID);
         }
+
+        [TestMethod]
+        public void ConditionMatchesExpectedElement()
+        {
+            int[] expectedTypes = { ControlType.Calendar, ControlType.Table };
+            var unexpectedTypes = ControlType.All.Difference(expectedTypes);
+            
+                var e = new MockA11yElement();
```

Comment:
This line is indented one tab too far
```suggestion
            var e = new MockA11yElement();
```